### PR TITLE
added jax.numpy.round function

### PR DESCRIPTION
**Pull request by x0axz**
Close https://github.com/unifyai/ivy/issues/12370

### CODE REVIEW

**Comment by Daniel4078** [comment]
Can you please comment `add_frontend_checklist` so that the Ivy bot can bring up a list of things to do in a frontend task, thanks!

**Comment by x0axz** [comment]
@Daniel4078 done. please review now.

File: .github/workflows/checklist_actions.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
   
   frontend_pr_commented:
     name: Frontend PR comment
-    if: ${{ github.event.issue.pull_request && github.event.comment.body == 'add_frontend_checklist' }}
+    if: ${{ github.event.issue.pull_request }}
```
Comment (Daniel4078) [inline .github/workflows/checklist_actions.yml:44]: I don't think you should push changes in this unrelated files. So can you revert it?

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 2]
```diff
@@ -2574,6 +2574,35 @@ def test_jax_numpy_around(
     )
 
 
+# round
+@handle_frontend_test(
+    fn_tree="jax.numpy.round",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=["float64"],
```
Comment (Daniel4078) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:2581]: why the available_dtype here is only one, you should use the helpers.get_dtypes() function instead to cover all cases.

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Found something for you to fix, check the comments below, thanks!

**Comment by Daniel4078** [comment]
I just found another thing for you to see: ivy/functional/frontends/jax/numpy/mathematical_functions.py:715:1: F811 redefinition of unused 'round' from line 545
It seems like there was another definition of round in the file, but that one may not be fully correct as well. So please remove that duplicated thing.

**Comment by x0axz** [comment]
@Daniel4078 Thanks for reviewing. Fixed mention issues; please review now.

**Review by Daniel4078 — APPROVED** [review]
There was a small backend problem(https://github.com/unifyai/ivy/actions/runs/5449648901/jobs/9914787471#step:3:3475). But it is not yours. Looks good to me, Thanks

**Pull request merged**